### PR TITLE
Background Based On Time

### PR DESCRIPTION
**Pull request by siddharthsinghvats**
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
A timer in which the background image changes according to time.

## Breaking changes
None

## Screenshots
None

### CODE REVIEW

File: src/blocks/siddharthsinghvats4/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
```
Comment (sophiabrandt) [inline src/blocks/siddharthsinghvats4/index.html:8]: Please add a descriptive title meta tag

Comment (siddharthsinghvats) [reply src/blocks/siddharthsinghvats4/index.html]: Added the title, thank you

**Comment by siddharthsinghvats** [comment]
Can it now be merged?

**Review by auxfuse — APPROVED** [review]
LGTM 🎉

**Pull request merged**